packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -163,7 +200,19 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
       return Math.max(this.props.min, Math.min(this.props.max, value));
     }
   }
-
+  private _getValueLabel(className: string): JSX.Element | null {
+    const { showValue, disabled, valueFormat, label } = this.props;
+
+    if (showValue) {
+      return (
+        <Label className={className} disabled={disabled}>
+          {label && ': '}
+          {valueFormat ? valueFormat(this.value!) : this.value}
```

Comment:
This value can be extracted before return like
`const value = valueFormat ? valueFormat(this.value!) : this.value;`
and later do something like this:
```ts
{ label ? `: ${value}` : value }
```

Reply:
ok great just added it